.github/workflows/pythonapp.yml

```diff
@@ -42,6 +42,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip wheel
+          python -m pip install monai==1.4.0rc11
```

Comment:
is this not part of requirements.txt?

Reply:
It is just a temp fix for the ci pass in this PR. Since monai still not release. `monai>=1.3.1` will not install rc version by default.